test/adapters/jasmine.html

```diff
@@ -2,23 +2,37 @@
 <html>
 	<head>
 		<title>Jasmine Spec Runner</title>
-		<link rel="shortcut icon" type="image/png" href="../../lib/jasmine/images/jasmine_favicon.png">
-		<link rel="stylesheet" type="text/css" href="../../lib/jasmine/lib/jasmine-core/jasmine.css">
+		<link rel="shortcut icon" type="image/png" href="../../node_modules/jasmine/node_modules/jasmine-core/images/jasmine_favicon.png">
+		<link rel="stylesheet" type="text/css" href="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.css">
 	</head>
 
 	<body>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine.js"></script>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/boot.js"></script>
 
 		<script src="../../node_modules/steal/steal.js"
 						main="funcunit/test/adapters/jasmine"></script>
 
 		<script>
 			steal.done().then(function() {
 				var jasmineEnv = jasmine.getEnv();
+			  var queryString = new jasmine.QueryString({
+			    getWindowLocation: function() { return window.location; }
+			  });
+			  
 				jasmineEnv.updateInterval = 1000;
 
-				var htmlReporter = new jasmine.HtmlReporter();
+
+				var htmlReporter = new jasmine.HtmlReporter({
+			    env: jasmineEnv,
+			    queryString: queryString,
+			    onRaiseExceptionsClick: function() { queryString.setParam("catch", !env.catchingExceptions()); },
+			    getContainer: function() { return document.body; },
+			    createElement: function() { return document.createElement.apply(document, arguments); },
+			    createTextNode: function() { return document.createTextNode.apply(document, arguments); },
+			    timer: new jasmine.Timer()
```

Comment:
The HTMLReporter object was just copied from [Jasmine's docs](http://jasmine.github.io/2.0/boot.html#section-Reporters).  I didn't look at the logic;  I was just trying to get this refactor to be in a solid state.